apps/src/studio/Item.js

```diff
@@ -335,6 +410,11 @@ Item.prototype.display = function () {
     opacity = Math.max(opacity, 0);
   }
 
+  // Watch behavior does not change logical position, should update every frame
+  if (this.activity === "watchActor") {
+    this.turnToFaceActor(0);
```

Comment:
You probably want to use `(Studio.protagonistSpriteIndex || 0)` for sprite index. While hoc2015 is only using one sprite, if we want to re-use this in other playlab cases, at least that gives the level builder a little more flexibility...